packages/components/src/components/table/table/table-styles.ts

```diff
@@ -1,41 +1,28 @@
-import {
-  getMediaQueryMin,
-  spacingStaticLarge,
-  spacingStaticMedium,
-  spacingStaticSmall,
-  textSmallStyle,
-} from '@porsche-design-system/utilities-v2';
+import { textSmallStyle, spacingFluidMedium } from '@porsche-design-system/utilities-v2';
 import { getCss } from '../../../utils';
-import {
-  addImportantToEachRule,
-  getFocusJssStyle,
-  getThemedColors,
-  hostHiddenStyles,
-  pxToRemWithUnit,
-} from '../../../styles';
+import { addImportantToEachRule, hostHiddenStyles, doGetThemedColors } from '../../../styles';
+import type { Theme } from '../../../types';
 
-const { primaryColor } = getThemedColors('light');
+export const cssVariableTableRowHoverColor = '--p-internal-table-row-hover-color';
+export const cssVariableTableBorderColor = '--p-internal-table-border-color';
+export const cssVariableTableTheme = '--p-internal-table-theme';
+export const getComponentCss = (theme: Theme): string => {
+  const { primaryColor, backgroundSurfaceColor, contrastLowColor } = doGetThemedColors(theme);
 
-export const getComponentCss = (): string => {
   return getCss({
     '@global': {
       ':host': addImportantToEachRule({
         display: 'block',
         ...hostHiddenStyles,
       }),
+      '::slotted(*)': addImportantToEachRule({
+        [cssVariableTableRowHoverColor]: backgroundSurfaceColor,
```

Comment:
I think we don't need this css variable if we would use `stateHoverColor` since it's identical on light and dark theme. Why have we used `backgroundSurfaceColor` at all?

Reply:
@stephanschroeter I asked Janick, why do we use backgroundSurfaceColor, and he replied: "Surface wäre tatsächlich der Wunsch, weil wir wir den table nicht auf Surface einsetzen. Der hover wert ist für produktiv nicht so geil weil ziemlich heavy." So we need this css-variable..